test/test_custom_ops.py

```diff
@@ -3364,6 +3364,110 @@ def _(x):
         with f.set_kernel_enabled("cpu", enabled=False):
             self.assertEqual(f(x), x + 1)
 
+    @skipIfTorchDynamo("Expected to fail due to no FakeTensor support; not a bug")
+    def test_register_vmap_kwargonly_low_level(self):
+        with torch.library._scoped_library("_torch_testing", "FRAGMENT") as lib:
+            lib.define("foo(Tensor x, *, float y) -> Tensor")
```

Comment:
Can you add a test that if we have a kwarg-only Tensor that we error out somewhere?

Reply:
added now